akka-docs/rst/scala/remoting-artery.rst

```diff
@@ -0,0 +1,453 @@
+transient network failures the failure detector is configured with a margin,
+``akka.remote.watch-failure-detector.acceptable-heartbeat-pause``. You may want to
+adjust the :ref:`remote-configuration-scala` of this depending on you environment.
+This is how the curve looks like for ``acceptable-heartbeat-pause`` configured to
+3 seconds.
+
+.. image:: ../images/phi3.png
+
+Serialization
+-------------
+
+When using remoting for actors you must ensure that the ``props`` and ``messages`` used for
+those actors are serializable. Failing to do so will cause the system to behave in an unintended way.
+
+For more information please see :ref:`serialization-scala`.
+
+ByteBuffer based serialization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+Disabling the Java Serializer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+Routers with Remote Destinations
+--------------------------------
+
+It is absolutely feasible to combine remoting with :ref:`routing-scala`.
+
+A pool of remote deployed routees can be configured as:
+
+.. includecode:: ../scala/code/docs/routing/RouterDocSpec.scala#config-remote-round-robin-pool
+
+This configuration setting will clone the actor defined in the ``Props`` of the ``remotePool`` 10
+times and deploy it evenly distributed across the two given target nodes.
+
+A group of remote actors can be configured as:
+
+.. includecode:: ../scala/code/docs/routing/RouterDocSpec.scala#config-remote-round-robin-group
+
+This configuration setting will send messages to the defined remote actor paths.
+It requires that you create the destination actors on the remote nodes with matching paths.
+That is not done by the router.
+
+.. _remote-sample-scala-artery:
+
+Remoting Sample
+---------------
+
+There is a more extensive remote example that comes with `Lightbend Activator <http://www.lightbend.com/platform/getstarted>`_.
+The tutorial named `Akka Remote Samples with Scala <http://www.lightbend.com/activator/template/akka-sample-remote-scala>`_
+demonstrates both remote deployment and look-up of remote actors.
+
+Performance tuning
+------------------
+
+Dedicated lane for large messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+External, shared Aeron media driver
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+Fine-tuning CPU usage latency tradeoff
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+Remote Configuration
+--------------------
+
+There are lots of configuration properties that are related to remoting in Akka. We refer to the
+:ref:`reference configuration <config-akka-remote>` for more information.
+
+.. note::
+
+   Setting properties like the listening IP and port number programmatically is
+   best done by using something like the following:
+
+   .. includecode:: ../java/code/docs/remoting/RemoteDeploymentDocTest.java#programmatic
+
+
+.. _remote-configuration-nat-artery:
+
+Akka behind NAT or in a Docker container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
it was just copy-pasted. Btw, you are again reviewing a merged PR. Necromancy? ;)